templates/zerver/emails/confirm_new_email.txt

```diff
@@ -1,13 +1,16 @@
-Hi!
+{{ _("Hi!") }}
 
+{% trans %}
 We received a request to change the email address for the Zulip account on
 {{ realm_uri }} from {{ old_email }} to {{ new_email }}. If you would like
-to confirm this change, please click this link:
-
+to confirm this change, please click below:
+{% endtrans %}
 {{ activate_url }}
 
+{% trans %}
 If you did not request this change, please contact us immediately at
 <{{ support_email }}>.
+{% endtrans %}
 
-Cheers,
-Team Zulip
+{% trans %}Cheers,
+Team Zulip{% endtrans %}
```

Comment:
I note this again below, but I think we should just tag each line of the signature block as its own translation tag; that saves 2x on the signature block strings.